packages/backend/src/graphql/schema.graphql

```diff
@@ -302,6 +304,7 @@ type PaymentQuote {
   minExchangeRate: Float!
   lowExchangeRateEstimate: Float!
   highExchangeRateEstimate: Float!
+  amountSent: UInt64!
```

Comment:
Why does this parameter need to be exposed on the API?

If it does need to be exposed, it should have a more descriptive name to ensure that the caller doesn't assume that it should be subtracted from (e.g.) the `maxSourceAmount`.

Reply:
It doesn't need to be exposed since it's in the `OutgoingPaymentOutcome`